src/components/Course/Content/ExerciseContent/index.js

```diff
@@ -114,6 +114,28 @@ const RenderContent = ({ data }) => {
     }
   }
   if (data.component === "table") {
+    // return tableData(data);
+    //Changing list data from row to column
+    const allData = data.value.map((item) => item.items);
+    const dataInRow = [];
+    for (const i in allData[0]) {
+      for (const j in allData) {
+        dataInRow.push(allData[j][i]);
+      }
+    }
+    const dataInCol = [];
+    for (const i in allData[0]) {
+      const temArray = [];
+      for (const j in dataInRow) {
+        if (j == 3) {
+          break;
+        } else {
+          temArray.push(dataInRow[j]);
+        }
+      }
+      dataInCol.push(temArray);
+      dataInRow.splice(0, 3);
+    }
```

Comment:
This could be done more succinctly as follows:
```suggestion
    const dataInCol = allData[0].map((_, i) => allData.map((_, j) => allData[j][i]));
```
This is basically what you came up with, but in lines 120-123, you essentially created a flattened version of this Array by pushing one element at a time and then undid the flattening in lines 127-138.